R/random-forest-development.R

```diff
@@ -124,16 +123,12 @@
 #' ,[A1CNBR]
 #' ,[GenderFLG]
 #' ,[ThirtyDayReadmitFLG]
-#' ,[InTestWindowFLG]
 #' FROM [SAM].[dbo].[HCRDiabetesClinical]
-#' WHERE InTestWindowFLG = 'N'
```

Comment:
Add comment about this query only pulling in training data (which has a label for each row)?